BD-3509 Add LINE message types (#8143)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/line/create/message_types.md

---
nav_title: Message Types
article_title: LINE Message Types
page_order: 1
description: "This article covers the different types of LINE messages."
page_type: reference
tool:
 - Campaigns
channel:
 - LINE
hidden: true
permalink: /line/create/message_types/
---

# LINE message types

> > This article covers the LINE message types you can compose, including aspects and li,itations, and is part of the LINE beta collection. [Return to the main page](https://www.braze.com/docs/line/).

Reviewer: ```suggestion
> This article covers the LINE message types you can compose, including aspects and limitations, and is part of the LINE beta collection. [Return to the main page]({{site.baseurl}}/line/).
```

{% alert important %}
LINE access is in beta and only available in select Braze packages. Reach out to your account manager or customer success manager to get started.
{% endalert %}

When you compose a LINE message, you can drag-and-drop message types into the composer and then customize them.

> ![][5]{: style="max-width:40%;"}

Reviewer 2: Be sure to add alt text to all images before you merge 👍🏽

## Text

A LINE text message can contain up to 5,000 characters.

Use cases include:
- Announce a limited-time promotion for clearance stock
- Send personalized birthday greetings with unique promotion cards
- Share quick updates about upcoming events

![][1]{: style="max-width:40%;"}

## Image

> A LINE image message can be added through the [Media Library]({{site.baseurl}}/user_guide/engagement_tools/templates_and_media/media_library/) or a URL.

Reviewer: We are no longer capitalizing media library! 

```suggestion
A LINE image message can be added through the [media library]({{site.baseurl}}/user_guide/engagement_tools/templates_and_media/media_library/) or a URL.
```

Use cases include:
- Showcase a vacation destination to inspire users to look into purchasing plane tickets
- Highlight end-of-season promotions to encourage users to stock up on next year's winter clothes with great deals
- Start a visual countdown to a storewide annual sale

![][2]{: style="max-width:40%;"}

### URL image

Use URL images for use cases that incorporate:

> - Liquid dynamic images

Reviewer 2: ```suggestion
- Liquid dynamic images by including the Liquid in your image source attribute. For example, you can insert {% raw %} `https://example.com/images/?imageBanner={{first_name}}` {% endraw %} as the image URL to include a user's first name in the image.
```

Reviewer 2: (Note: typically i try to avoid the multi-nested bullets if possible because i feel it can be a bit busy).

>     - Include the Liquid in your image source attribute. For example, you can insert {% raw %} `https://example.com/images/?imageBanner={{first_name}}` {% endraw %} as the image URL to include a user's first name in the image. 

Reviewer 2: ```suggestion
```

- [Connected Content]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/connected_content/)
- [Braze catalogs]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/catalogs/)

| **Image specifications** | **Recommended properties** |
|--------------------------|----------------------------|
| Image file URL length | 2,000 characters maximum  |
| Image format          | PNG, JPEG             |
| File size     |  10&nbsp;MB maximum |
{: .reset-td-br-1 .reset-td-br-2}

## Rich messages (image map)

> A LINE rich message is an image that contains one or more links that are opened by selecting specific areas on the image. Select a rich message templates to choose how the links are mapped onto the image.

Reviewer: @rachel-feinberg can we add descriptions like this one to each of the message types? We need to make sure that the differences are clear even without the context of the pictures.
